recipes/harfbuzz/all/conanfile.py

```diff
@@ -82,9 +82,9 @@ def requirements(self):
         if self.options.with_freetype:
             self.requires("freetype/2.13.2")
         if self.options.with_icu:
-            self.requires("icu/74.1")
+            self.requires("icu/74.2")
```

Comment:
qt is the only one running 74.2 for now - maybe this could be delayed until all others also get bumped? (Chekcing if it makes sense to allow icu version ranges meanwhile!)